cluster-scope/base/external-secrets.io/externalsecrets/open-cluster-management-observability-alertmanager-config/externalsecret.yaml

```diff
@@ -0,0 +1,58 @@
+apiVersion: external-secrets.io/v1beta1
+kind: ExternalSecret
+metadata:
+  name: open-cluster-management-observability-alertmanager-config
+  namespace: open-cluster-management-observability
+spec:
+  refreshInterval: 15s
+  secretStoreRef:
+    name: nerc-cluster-secrets
+    kind: ClusterSecretStore
+  target:
+    name: alertmanager-config
+    template:
+      type: Opaque
+      engineVersion: v2
+      data:
+        alertmanager.yaml: |
+          global:
+          route:
+            receiver: default
+            group_by: [cluster, alertname]
+            routes:
+              - receiver: slack-notifications
+                group_by: [alertname, datacenter, app]
+                group_wait: 1d
+                group_interval: 1d
+                repeat_interval: 1d
+                matchers:
+                  - alertname = StoragePersistentVolumeFillingUp
+          receivers:
+          - name: default
+          - name: slack-notifications
+            slack_configs:
+            - channel: '#alerts-nerc-ocp'
+              api_url: "{{ .slack_api_url }}"
+              mrkdwn_in:
+                - text
+                - title
+              text: |{{ .slack_config_text }}
+              title: "{{ .slack_config_title }}"
+              title_link: "{{ .slack_config_title_link }}"
+  data:
+  - secretKey: slack_api_url
+    remoteRef:
+      key: nerc/nerc-ocp-infra/open-cluster-management-observability/slack/alerts-nerc-ocp
```

Comment:
This external secret has a bunch of things that make it environment specific (channel name, the secret). Though we'll probably use the same channel for alerts for both clusters anyway. I would probably put this directly in an overlay but I wouldn't block the PR on that.